librad/src/git/types.rs

```diff
@@ -224,28 +282,51 @@ impl Refspec<PeerId, PeerId> {
             {
                 let their_singed_rad_refs = rad_signed_refs_of(tracked_peer)?;
                 for (name, target) in their_singed_rad_refs.heads {
-                    let name_namespaced = format!("refs/namespaces/{}/{}", namespace, name);
-                    if let Some(name) = name.strip_prefix("refs/heads/") {
```

Comment:
We were stripping the `refs/heads` before but it doesn't look like we're doing that in the new changes. Could be that we're ending up with `refs/heads/refs/heads/foobar`

Reply:
`name` is of type `OneLevel` now -- no more strippin

Reply:
All the clubs are shut down at this level, strippin is verboten